src/main/java/com/timeaway/controllers/PaidTimeOffController.java

```diff
@@ -0,0 +1,36 @@
+package com.timeaway.controllers;
+
+import com.timeaway.repositories.PaidTimeOff;
+import com.timeaway.service.PaidTimeOffService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/paid-time-off")
+public class PaidTimeOffController {
+    private final PaidTimeOffService paidTimeOffService;
+
+    public PaidTimeOffController(PaidTimeOffService paidTimeOffService) {
+        this.paidTimeOffService = paidTimeOffService;
+    }
+
+    @GetMapping("/all")
+    public List<PaidTimeOff> getAllPto() {
+        return paidTimeOffService.getAllPto();
+    }
+
+    @GetMapping("/{id}")
+    public PaidTimeOff findPtoById(@PathVariable Long id) {
+        return paidTimeOffService.findPtoById(id);
+    }
+
+    @PostMapping("/create")
```

Comment:
Usually when creating such endpoints the following format for paths is used:
- POST /pto - create a new entity
- GET /pto - get all entities
- GET /prot/{id} - get entity by ID

I suggest to use that format.